bin/node/runtime/pangolin/src/pallets/evm_.rs

```diff
@@ -32,10 +31,7 @@ impl<R: dvm_ethereum::Config> PrecompileSet for PangolinPrecompiles<R> {
 			_ if address == to_address(4) => Some(Identity::execute(input, target_gas, context)),
 			// Darwinia precompiles
 			_ if address == to_address(21) => {
-				Some(<WithDraw<R>>::execute(input, target_gas, context))
-			}
-			_ if address == to_address(22) => {
-				Some(<KtonPrecompile<R>>::execute(input, target_gas, context))
+				Some(<Transfer<R>>::execute(input, target_gas, context))
 			}
 			_ if address == to_address(23) => {
 				Some(<Issuing<R>>::execute(input, target_gas, context))
```

Comment:
```Issuing``` might need to be replaced by ```Dispatch``` Precompile, am I right?  @xiaoch05 @AsceticBear

Reply:
Yes, later it will be replaced.

Reply:
In my understanding, you are right! But there will be a `dispatch_wapper precompile` to replace the current `Issuing` contract.

Reply:
The Dispatch Precompile Wrapper coding will be completed with s2s backing & issuing PR, so in this pr, we keep it as the old version.